### DISCUSSION TITLE
Accessing data outside scope on v3+

### DISCUSSION THREAD

**Discussion by dawsonnn3** [alpinejs/alpine#2149] [1. Help]
I'm currently in the process of moving everything over from v2 to v3.

When I wanted to access or change data outside of its scope, I done something like this:

```
<div id="popup" x-data="{show:true}">
document.getElementById('popup').__x.$data.show = true;
```

Fairly simple stuff. However, this no longer works in v3. I can't see anything regarding this or any other way of accessing data outside of its scope on the docs. Is there another way to do this?

**Comment by SimoTod** [comment]
It would be better to use a store in this case. It wasn't public api in v2 and it was not really human friendly anyway.
